pmm/v3/aws-staging-start-pmm3.groovy

```diff
@@ -0,0 +1,477 @@
+import hudson.model.Node.Mode
+import hudson.slaves.*
+import jenkins.model.Jenkins
+import hudson.plugins.sshslaves.SSHLauncher
+
+library changelog: false, identifier: 'lib@PMM-12557', retriever: modernSCM([
```

Comment:
Normally, this is only necessary when you need to pull from a specific branch hosting "vars" aka "shared function library". Since this PR does not modify them, let's revert the last commit.